### PR TITLE
Make healium be as useful as cryoxadone

### PR DESCRIPTION
**Pull request by warface1234455**
# Document the changes in your pull request
Healium can now heal cloneloss
Healium can now treat all type of wounds (Only work during unconscious and below 0C)
Healium now remove trait_disfigured
Healium can now preserve organ if metabolizing
Healium can now restore blood (Only work during unconscious and below 0C)

# Why is this good for the game?
So called healium supposed to heal other type of wounds as well right? More useful in cryotube and this stuffs is quite not that easy to make so would be fair to make it worth for all the efforts 

# Testing

https://github.com/yogstation13/Yogstation/assets/89688125/85ed543c-cf7b-4e52-9384-780c5108cc79

# Wiki Documentation
in document of change

# Changelog

:cl:  
tweak: Healium can now heal cloneloss
tweak: Healium can now treat all type of wounds (Only work during unconscious and below 0C)
tweak: Healium now remove trait_disfigured
tweak: Healium can now preserve organ if metabolizing
tweak: Healium can now restore blood (Only work during unconscious and below 0C)
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -253,19 +253,29 @@
...
+	var/datum/gas_mixture/air1 = airs[1]
+
 	if(mob_occupant.health >= mob_occupant.getMaxHealth() - robotic_limb_damage) // Don't bother with fully healed people. Now takes robotic limbs into account.
-		var/has_cryo_wound = FALSE
+		var/has_wound = FALSE
 		if(C && C.all_wounds)
-			for(var/datum/wound/wound as anything in C.all_wounds)
-				if(wound.wound_flags & ACCEPTS_CRYO)
+			if(air1.total_moles() && air1.get_moles(GAS_HEALIUM) > MINIMUM_MOLE_COUNT)
+				if(C.all_wounds)
```
Comment (Runian) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Redundant check for `C.all_wounds`? Was just checked two lines ago (`C && C.all_wounds`).

Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Bruh

Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Didn't realize lmao

Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Imma fix this tmr

Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Its 3ambtw

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**